homeassistant/components/jewish_calendar/entity.py

```diff
@@ -1,18 +1,27 @@
 """Entity representing a Jewish Calendar sensor."""
 
-from typing import Any
+from dataclasses import dataclass
+
+from hdate import Location
 
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.const import CONF_LANGUAGE, CONF_LOCATION
 from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
 from homeassistant.helpers.entity import Entity, EntityDescription
 
-from .const import (
-    CONF_CANDLE_LIGHT_MINUTES,
-    CONF_DIASPORA,
-    CONF_HAVDALAH_OFFSET_MINUTES,
-    DOMAIN,
-)
+from .const import DOMAIN
+
+type JCalConfigEntry = ConfigEntry[JCalData]
```

Comment:
I think it's cleaner to keep the full prefix "JewishCalendar"
```suggestion
type JewishCalendarConfigEntry = ConfigEntry[JewishCalendarData]
```